catalog/datasets/aat.jsonld

```diff
@@ -25,10 +25,23 @@
       "@type": "DataDownload",
       "contentUrl": "http://vocab.getty.edu/sparql",
       "encodingFormat": "application/sparql-query",
-      "potentialAction": {
-        "@type": "SearchAction",
-        "query": "file://catalog/queries/aat.rq"
-      }
+      "potentialAction": [
+        {
+          "@type": "SearchAction",
+          "query": "file://catalog/queries/search/aat.rq"
+        },
+        {
+          "@type": "FindAction",
+          "query": "file://catalog/queries/lookup/aat.rq",
+          "identifier": [
+            {
+              "@type": "PropertyValue",
+              "propertyID": "URI Prefix",
+              "value": "http://vocab.getty.edu/aat/"
```

Comment:
To discuss: is there a better way to refer to the URI prefix? I have now used a `PropertyValue` type with our own `propertyID` value `URL Prefix`.

Reply:
See above.